src/ucm/util/sys.c

```diff
@@ -33,7 +33,8 @@ ucm_global_config_t ucm_global_opts = {
     .enable_cuda_reloc          = 1,
     .enable_dynamic_mmap_thresh = 1,
     .alloc_alignment            = 16,
-    .enable_syscall             = 0
+    .enable_syscall             = 0,
+    .enable_bistro_hook         = 0
```

Comment:
let's make bistro and syscalls enabled by default

Reply:
enabled by default